cmd/juju/caas/add.go

```diff
@@ -128,6 +126,21 @@ func (c *AddCAASCommand) newAPIRoot() (api.Connection, error) {
 	return c.NewAPIRoot()
 }
 
+func getStdinPipe(ctxt *cmd.Context) io.Reader {
```

Comment:
maybe a comment is needed - you and i know what it does but a casual reader may not